src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CsvDataExtractionServiceTest.kt

```diff
@@ -242,4 +268,195 @@ class CsvDataExtractionServiceTest {
       "eventId,timestamp,videoLinkBookingId,eventType,agencyId,probationTeam,probationTeamId,madeByProbation,mainStartTime,mainEndTime,preStartTime,preEndTime,postStartTime,postEndTime,mainLocationName,preLocationName,postLocationName,meetingType,user\n" +
       "1,2024-07-01T09:00:00,1,CREATE,$RISLEY,\"probation team description\",\"probation code\",true,2024-07-10T10:00:00,2024-07-10T11:00:00,,,,,\"${risleyLocation.localName}\",,,\"Pre-sentence report\",probation_user\n"
   }
+
+  @Test
+  fun `should produce CSV for prison room configuration`() {
+    whenever(courtsService.getCourts(eq(true))) doReturn listOfCourts
```

Comment:
Not sure the `eq(true)` is necessary on a few of this lines.  Does no harm though.